tests/call/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -602,23 +602,21 @@ string
 string
 string');
 
-$a
-    ->call(
-        $a,
-        '
+$a->call(
+    $a,
+    '
 string
 string
 string',
-        $c
-    )
-    ->call(
-        $a,
-        '
+    $c
+)->call(
+    $a,
+    '
```

Comment:
I think it is not good idea, ugly, prettier@2 improve this, we should copy/paste logic from prettier :smile:

Reply:
Seems that prettier is doing the same: [playground](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBDAdAIwBQEYA0ABAAYA6UAZhBOZqgE62oBexRATAJTph5HFUaUOvWKcQ+EBAAOMAJbQAzslAN6EAO4AFBgmUpUAGw2oAnssmZ6qMAGs4MAMrSbcqAHNkMegFc4kgAsYAFtDAHUAuXhFFzA4Rz0ouQA3KNNkcEULEDdFOHoYLWt3YNRkCiM8yQArRQAPACFrOwdHVGC4ABk3OHLK-xBausc3d0M4AEUfCHg+wyqQF3o8+gy6TDhDCUX6NxgwuQATGADkAA4ABklpdTyw62kMm7gV5N7JAEdp+CKZfTRFABaKBwOCHMHbehwL5yKFFVAlMpICrzAZ5YJyLy+NGjcZTGa9ZH9SQwVCYA7HU5Idgk6xyQyjADCEGCpQyLwArNsfHkACpk-QohbJPwASSg4NgjjAu1kAEEJY4YKZxnM8gBfdVAA)

Reply:
Friendly ping @evilebottnawi 🙃